hack/lib.bash

```
@@ -0,0 +1,364 @@
+        kubectl port-forward svc/frontproxy-front-proxy "$port:6443" 2>/dev/null >/dev/null &
+}
+
+kcp::create_workspace() {
+    local parent_kubeconfig="$1"
+    [[ -z "$parent_kubeconfig" ]] && die "parent_kubeconfig is required"
+    local target_kubeconfig="$2"
+    [[ -z "$target_kubeconfig" ]] && die "target_kubeconfig is required"
+    local wsname="$3"
+    [[ -z "$wsname" ]] && die "wsname is required"
+
+    local current_server="$(kubeconfig::hostname "$parent_kubeconfig")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)